src/optionDefaults.js

```diff
@@ -0,0 +1,110 @@
+export const optionDefaults = {
+  debug: false,
+  views: {
+    home: {
+      order: 1,
+      hidden: false,
+    },
+    light: {
+      order: 2,
+      hidden: false,
+    },
+    fan: {
+      order: 3,
+      hidden: false,
+    },
+    cover: {
+      order: 4,
+      hidden: false,
+    },
+    switch: {
+      order: 5,
+      hidden: false,
+    },
+    climate: {
+      order: 6,
+      hidden: false,
+    },
+    camera: {
+      order: 7,
+      hidden: false,
+    }
+  },
+  areas: {
+    undisclosed: {
```

Comment:
I think this undisclosed should be renamed to unassigned. In HA, you assign areas to devices and other things, so devices without areas can be called unassigned devices.
What do you think?

Reply:
I get what your saying, but...

In this case, the point of view is from <ins>area</ins> and not from <ins>device</ins>.
The area itself is "undisclosed", meaning "not made known or revealed".

We have <ins>area</ins>-cards and entities grouped by <ins>area</ins>.
Calling an area "Unassigned Devices" is ambiguous; an area isn't a device (or group of devices) after all.
Also, what would "unassigned" mean? Unassigned from what?

Reply:
I understand what you mean with assigned, but anyone else not following this discussion who encounters the term "unassigned devices" most likely won't. ;)

If you agree, we keep it undisclosed for now, we can change the area name later on if the need is there.
Also, you can configure the name in the yaml, just like with any other area.